tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -6083,9 +6149,12 @@ <h2>Show similarity to selected datapoint</h2>
             const feature = featureStats.name;
             this.distanceStats_[feature] = {};
             if (featureStats.numStats) {
-              // For numeric features, store standard deviation.
-              this.distanceStats_[feature].stdDev =
-                featureStats.numStats.stdDev;
+              // For numeric features, store standard deviation, min and max.
```

Comment:
Not sure if this comment is adding any information that is not present in the code below. Consider removing it.

Reply:
I dropped the redundant part, but do think it's important to say that this block handles numeric features, as opposed to `For categorical features...` in the following block.